ocaml/xapi/xapi_globs.ml

```diff
@@ -791,6 +791,8 @@ let list_domains = ref "/usr/bin/list_domains"
 
 let xen_cmdline_script = ref "/opt/xensource/libexec/xen-cmdline"
 
+let alert_certificate_check = ref "/opt/xensource/libexec/alert-certificate-check"
```

Comment:
Drop the `/opt/xensource/libexec/` part, and make it just `ref "alert-certificate-check"`. See my related comment. The "examples" above are not right.